functions/Get-DbaAgentProxy.ps1

```diff
@@ -41,6 +44,7 @@
         [Alias("ServerInstance", "Instance", "SqlServer")]
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]$SqlCredential,
+        [object[]]$proxy
```

Comment:
You can change this to [string[]]. We did object for a long time but now are moving to string.